### PR TITLE
Document contributing across versions

### PR DESCRIPTION
**Pull request by jdbaldry**
Signed-off-by: Jack Baldry <jack.baldry@grafana.com>

### CODE REVIEW

File: docs/sources/writing-guide/contribute-documentation/index.md [hunk 1]
```diff
@@ -116,3 +116,13 @@ When you are ready for other people to review your work, perform the following t
 When you add a PR to the repo and assign the `type/docs` label, it will be reviewed by a member of the Docs Squad.
 
 Depending on the size of the PR and the priority of other work, the PR will either be immediately reviewed and merged (minor fixes typically follow this pattern) or the PR will be triaged and placed in the backlog of work or moved into further development.
+
+## Contributing across versions
+
+Editing the main branch of a project affects the content in the "next" directory of the website.
+To edit previous versions including "latest", the most recent release, and other versions, you must backport the changes into the long-lived version branches in the project repository.
```
Comment (osg-grafana) [inline docs/sources/writing-guide/contribute-documentation/index.md:123]:
```suggestion
When you edit the `main` branch of a project, it affects the content in the `next` directory of the website.
To edit a previous version, or `latest` (the most recent release), you must backport the changes into the long-lived version branches in the project repository.
```
Does backporting always relate to a previous release?

File: docs/sources/writing-guide/contribute-documentation/index.md [hunk 2]
```diff
@@ -116,3 +116,13 @@ When you are ready for other people to review your work, perform the following t
...
+
+Backporting is most easily achieved using the backport vMAJOR.MINOR.x labels on the GitHub pull request.
```
Comment (osg-grafana) [inline docs/sources/writing-guide/contribute-documentation/index.md:125]:
```suggestion
To backport a change, use the `backport <vMAJOR.MINOR.x>` labels on the GitHub pull request.
```

File: docs/sources/writing-guide/contribute-documentation/index.md [hunk 3]
```diff
@@ -116,3 +116,13 @@ When you are ready for other people to review your work, perform the following t
...
+
+`grafanabot` automatically creates a backport pull request after the original pull request is merged if the merge commit can be cherry-picked without conflict.
```
Comment (osg-grafana) [inline docs/sources/writing-guide/contribute-documentation/index.md:127]:
```suggestion
`grafanabot` automatically creates a backport pull request after the original pull request is merged, if the merge commit can be cherry-picked without a conflict.
```

**Review by osg-grafana — APPROVED** [review]
Unblocking with feedback

File: docs/sources/writing-guide/contribute-documentation/index.md (hunk 3 above)
Comment (jdbaldry) [reply docs/sources/writing-guide/contribute-documentation/index.md:127]: Can we keep "cherry-picked" as it matches with the `git` subcommand (`git cherry-pick`) or perhaps should that dash indicate that I should use "cherry picked"?

File: docs/sources/writing-guide/contribute-documentation/index.md (hunk 1 above)
Comment (jdbaldry) [reply docs/sources/writing-guide/contribute-documentation/index.md:123]: > Does backporting always relate to a previous release?

Good catch, there could also be a version branch for the next-to-be-released version, I'll need to rewrite this sentence.

**Review by alyssawada — APPROVED** [review]
Including @osg-grafanas feedback, looks good!

File: docs/sources/writing-guide/contribute-documentation/index.md (hunk 3 above)
Comment (osg-grafana) [reply docs/sources/writing-guide/contribute-documentation/index.md:127]: Yes, definitely. Nice catch.

**Comment by osg-grafana** [comment]
Can this be merged?

**Comment by osg-grafana** [comment]
@jdbaldry, can this be moved along?

**Pull request merged**